common/fs.cpp

```diff
@@ -362,6 +362,18 @@ void FSDirectory::cacheDirectoryRecursive(FSNode node, int depth, const Path& pr
 				}
 			} else {
 				_fileCache[name] = *it;
+
+#ifdef MACOSX
+				// On Mac, check for native resource fork
+				String rsrcName = it->getPath() + "/..namedfork/rsrc";
+				FSNode rsrc = FSNode(rsrcName);
+
+				Path cacheName = prefix.appendComponent(it->getRealName() + "/..namedfork/rsrc");
```

Comment:
As mentioned by @lephilousophe on Discord, this should better use `join()` instead of `appendComponent()`